website/www/site/i18n/home/en.yaml

```
@@ -33,4 +33,18 @@
 - id: home-section
   translation: "section"
 - id: home-ctas-title
-  translation: "Get started"
\ No newline at end of file
+  translation: "Get started"
+- id: home-model-title1
+  translation: "How it works?"
```

```suggestion
  translation: "How Does It Work?"
```

Gris told me offline that this is supposed to say "How Does it work?", does the copy in this file need to be updated still?